Automattic/jetpack
# Editor: Add WordPress.com integrations to the block editor

Fixes https://github.com/Automattic/wp-calypso/issues/32044.
Background: p3fqKv-6Hh-p2.

### Changes proposed in this Pull Request:

Load the scripts of the [WordPress.com block editor integrations](https://github.com/Automattic/wp-calypso/tree/master/apps/wpcom-block-editor) that enhances the block editor with the features available in WordPress.com sites

**Calypso Iframed block editor:**
- URL is updated with the draft ID when the post is saved.
- Trashing a post redirects to the list of trashed posts in Calypso displaying a success notification.
- Revisions are managed with the Calypso revisions modal.
- Reblogging a post from the Calypso reader creates a post in Calypso with the except title and content.
- Media items are managed with the Calypso media modal.
- Posts are previewed with the Calypso preview modal.
- Back button on the header ("View posts") redirects to the list of posts in Calypso.
- Media items inserted in classic blocks are managed with the Calypso media modal.

**WP Admin block editor** (also included in the iframed block editor):
- Support for the underline and justified text formats.

#### Testing instructions:
* Start Calypso locally using the `update/me-sites-version` branch (https://github.com/Automattic/wp-calypso/pull/32527).
* Go to `/block-editor/post/:yourJetpackDomain`.
* Verify all the integrations listed above work as expected.
* Load `:yourJetpackDomain/wp-admin/post-new.php`.
* Check the underline and justified text formats are available on the paragraph blocks. 

#### Proposed changelog entry for your changes:
I guess it's not needed?
